parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -125,6 +125,8 @@ fi
 
 logs_to_events "AKS.CSE.disableSystemdResolved" disableSystemdResolved
 
+# ensureSSHService has been moved to VHD build phase to improve node provisioning performance
```

Comment:
where is `ensureSSHService` getting removed ?  I only see the new comment getting added.

Is there a configuration that we would want to be configured at bootstraptime for the SSH service ?